### DISCUSSION TITLE
关于在升级1.6.0版本时出现的问题

### DISCUSSION THREAD

**Discussion by yeelvus** [xaoxuu/hexo-theme-stellar#57] [Q&A] [answered]
**请问应该如何升级到最新版本嘞？**

我是从1.4.2升级到1.5.2版本的，升级时没有问题，但准备升级到1.6版本时就出现这个错误了。
还不知到如何解决。是有其他的相关的组件要一起升级吗？

![Snipaste_2022-01-23_22-17-06](https://user-images.githubusercontent.com/50106230/150719847-84387ed7-f6e0-4ae5-893a-45c2b079b9f0.png)

**Comment by xaoxuu** [comment, marked as answer]
https://github.com/xaoxuu/hexo-theme-stellar/releases
建议看着更新日志进行升级，报错位置是因为类型改了：
![image](https://user-images.githubusercontent.com/16400144/150727976-4af73c47-ce94-4ca9-bce3-35ea4118f133.png)
